exasol_integration_test_docker_environment/lib/test_environment/spawn_test_environment_with_external_db.py

```diff
@@ -31,7 +34,9 @@ def create_network_task(self, attempt: int):
                 network_name=self.network_name
             )
 
-    def create_spawn_database_task(self, network_info: DockerNetworkInfo, attempt: int):
+    def create_spawn_database_task(self, network_info: DockerNetworkInfo,
+                                   certificate_volume_info: Optional[DockerVolumeInfo], attempt: int):
+        assert certificate_volume_info is None
```

Comment:
Don't forget assert can be deactivated and are probably by default

Reply:
Done